src/ssa_ir/converter/insn_converter.rs

```diff
@@ -0,0 +1,443 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
The naming has multiple issues.

"ssa_ir" is not great. 
I think it needs a better name. We have existing CFG format which is the higher-level format. Maybe should call this LIR (low-level IR) and CFG should be renamed HIR.  The CFG -> HIR rename is not something for this PR.
"converter" should probably lowering, that describes it better
"ins_converter" repeats the converter part which is alreader part of its namespace. "insn" is too shorter. How about calling this "instructions.rs".